us_election_lukasz.sql

```diff
@@ -0,0 +1,223 @@
+-- najwi�kszy odsetek g�os�w Missisipi, Alabama, South Carolina, Virginia, Georgia, Alabama, Tennessee, 
+select state, county, votes, fraction_votes from primary_results pr 
+where candidate ilike '%Clinton' 
+group by 1, 2, 3 , 4
+order by 4 desc 
+
+--Najwieksza liczba glosow California, Floryda, New York, Illinois, Texas
+select state, sum(votes) as liczba_glosow from primary_results pr 
+where candidate ilike '%Clinton'
+group by 1
+order by 2 desc 
+
+--Najmniejsza liczba glosow Alaska, North Dakota, Wyoming, Maine, Idaho
+select state, sum(votes) as liczba_glosow from primary_results pr 
+where candidate ilike '%Clinton'
+group by 1
+order by 2  
+
+-- Suma glosow calkowita Hilary w USA
+select sum(votes) as liczba_calkowita_glosow from primary_results pr 
+where candidate ilike '%Clinton' -- 15 692 452 glosy
+
+--liczba glosow ca�kowita w USA
+select sum(votes) from primary_results pr --56 759 187 glosy
+
+--ilosc miast z P25, P50 , P75
+select percentile_disc(0.25) within group (order by fraction_votes ) from primary_results pr 
+where candidate ilike '%Clinton' -- 0.356
+
+select count(county) from primary_results pr 
+where candidate ilike '%Clinton' and fraction_votes < 0.356 --1 046 miast, gdzie jej udzial by� <25%
+
+select percentile_disc(0.50) within group (order by fraction_votes ) from primary_results pr 
+where candidate ilike '%Clinton' -- 0.47
+
+select count(county) from primary_results pr 
+where candidate ilike '%Clinton' and fraction_votes < 0.47 and fraction_votes >= 0.356 --1 055 miast, gdzie jej udzial by� >25% i <50%
+
+select percentile_disc(0.75) within group (order by fraction_votes ) from primary_results pr 
+where candidate ilike '%Clinton' -- 0.577
+
+select count(county) from primary_results pr 
+where candidate ilike '%Clinton' and fraction_votes < 0.577 and fraction_votes >= 0.47 --1 049 miast, gdzie jej udzial by� >50% i <75%
```

Comment:
Rozumiem, że wartości 0.577 oraz 0.47 są wynikiem poprzednich zapytań. 
Spróbuj je zamienić na zmienne, tak aby można było korzystać z tego kodu na wypadek zmian wartości, dopisania kolejnych rekordów do tabeli. Można to wykonać za pomocą podzapytania lub zapisać wyniki poprzednich zapytań do tabel i je połączyć. 
W razie wątpliwości zapraszam do kontaktu.

Reply:
Próbowałem na kilka sposobów - ale bez rezultatu